README.md

```diff
@@ -84,7 +84,12 @@ Tests are an important part of the development and evolution of this project,
 and new contributions are expected to include tests for any functionality
 change.  To run the tests, pass the `test` verb to the `bootstrap` script:
 
-    ./Utilities/bootstrap --build-tests test
```

Comment:
Does the old version work? I was unable to get `Utilities/bootstrap` to succeed without passing in `--swiftc`, `--sbt`, and `--xctest`. I think we should include these in the README today, not just when this pull request is merged.

I may be missing something, though. Is it possible to successfully run *just* `Utilities/bootstrap test`?